tester_app/client/webpack.client.hot.config.js

```diff
@@ -0,0 +1,63 @@
+// Run like this:
```

Comment:
// This config file setups up the Webpack Dev Server: https://webpack.github.io/docs/webpack-dev-server.html